supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemAsyncJobExecutionActor.scala

```diff
@@ -220,14 +220,19 @@ trait SharedFileSystemAsyncJobExecutionActor
   }
 
   private def metadataKey(key: String) = MetadataKey(workflowId, Option(metadataJobKey), key)
-
```

Comment:
ToL: You can try to remove these newlines separating `def`s, but our current IntelliJ formatter (the default) will keep adding them back.